land_grab_2/stl_dataset/step_4/dataset_summary_stats.py

```diff
@@ -79,7 +79,16 @@ def tribe_summary_for_univ_summary(df, col_filter_func, out_column_name):
 
 
 def combine_cession_ids(v):
-    raw_nums = set(itertools.chain.from_iterable([c.split(',') if ',' in c else c.split(' ') for c in v.tolist() if c]))
```

Comment:
@mariaparazorose did you mean to include these edits in this PR?

Reply:
These were edits from a previous fix I had made when I sent Parker the output of 2_5 and, after testing it out on steps 3 and 4, was getting errors on how the cession information was being generated.